xmake/core/tool/toolchain.lua

```diff
@@ -288,12 +288,18 @@ function _instance:configs_save()
 end
 
 -- do check, we only check it once for all architectures
-function _instance:check()
-    local checked = self:config("__checked")
+function _instance:check(opt)
+
+    opt = opt or {}
+    local checked_config = "__checked"
+    if opt.ignore_sdk then
```

Comment:
why?

Reply:
because when --sdk= is set for llvm, it make msvc toolchain check try to find vc build tools in llvm sdk
so it prevent xmake to find msstl std module with llvm when using --sdk=

Reply:
i found a better fix